src/Microsoft.ML.Data/DataLoadSave/Binary/Codecs.cs

```diff
@@ -1051,11 +1051,8 @@ public Reader(VBufferCodec<T> codec, Stream stream, int items)
                     // Get a buffer.
                     var values = codec._bufferPool.Get();
                     Utils.EnsureSize(ref values, totalItems, false);
-                    if (totalItems > 0)
```

Comment:
>totalItems [](start = 24, length = 10)

does it need to throw if it is not reading anything here?

Reply:
Oh, it will, but inside reader.


---
In reply to: [234873196](https://github.com/dotnet/machinelearning/pull/1670#discussion_r234873196) [](ancestors = 234873196)